### DISCUSSION TITLE
How to create a voice bank for OpenUtau?

### DISCUSSION THREAD

**Discussion by ElBlandersito** [stakira/OpenUtau#1258] [Q&A]
Hello, I would like to know how to create a voice bank for OpenUtau (especially one in Spanish). I have reviewed the Wiki but I have not fouund anything. Can you give me information?

**Comment by BagelHero** [comment]
OpenUtau voicebanks are just UTAU voicebanks! But, there is a certain page that appears hidden at first glance-- we'll make it more obvious.
https://github.com/stakira/OpenUtau/wiki/Voicebank-development

**Reply by ElBlandersito** [reply]
thank you
